examples/mnist_export_v2/export_test.py

```diff
@@ -0,0 +1,57 @@
+import tensorflow as tf
+import tensorflow_hub as hub
+import unittest
+import tempfile
+import os
+from absl.testing import absltest
+import export
+
+model = None
+TEST_DATA_DIR = "examples/mnist_export_v2/test_data"
+
+def load_model(path):
+  global model
+  if not model:
+    model = hub.load(path)
+  return model
+
+
+class TFHubMNISTTest(tf.test.TestCase):
+  def setUp(self):
+    file_ = tf.gather(tf.io.matching_files("%s/*.jpg" % TEST_DATA_DIR), 0)
+    self.test_image = tf.expand_dims(
+        tf.image.decode_jpeg(
+            tf.io.read_file(file_)), 0)
+    self.test_label = int(str(file_.numpy()).split("/")[-1].split(".jpg")[0])
+
+  def test_model_exporting(self):
+    mocked_dict = export.sys.__dict__.copy()
+    with open(os.devnull, "w") as devnull:
```

Comment:
Can you test the exporting behaviour without mocking?

Reply:
Yep, I can actually! But, I mocked it to prevent printing anything. Sachin said, while unittesting it is better to not print anything. Is it okay to print out the training steps while testing?

Reply:
I see. I think printing is not ideal, but still better than unreadable code. You can make it train for minimal number of steps in the test scenario, so there won't be much printing.

Reply:
Okay! On it! This should just print one line. As it is clearing out the previous line using `\r` :)